v22.2/sso-sql.md

```diff
@@ -0,0 +1,88 @@
+	
+		- A SQL user specifically corresponding to the service account must be pre-provisioned on the cluster (or you must have access to a SQL role allowing you to create such a user).
+
+## Configure your cluster settings
+In order to authenticate a service account to a {{ site.data.products.db }} cluster using a JWT issuer, you must update several cluster settings in the `server.jwt_authentication` namespace:
+
+1. `server.jwt_authentication.enabled`
+
+	JWT authentication must be enabled on your cluster.
+
+	Required value: `true`
+
+1. `server.jwt_authentication.jwks`
+
+	Add your IdP's public signing key to your cluster's list of accepted signing JSON web keys (JWKS), under the `jwks` setting.
+
+	This must be a [JWK](https://www.rfc-editor.org/rfc/rfc7517) formatted single key or key set, containing the [public keys](../{{site.versions["stable"]}}/security-reference/transport-layer-security.html#key-pairs) for SSO token issuers/IdPs that will be accepted by your cluster.
+
+1. `server.jwt_authentication.issuers`
+
+	Add your IdP's formal `issuer` name (this must match the `issuer` field in the JWT itself) to your cluster's list of accepted token issuers:
+
+	A comma separated list of formal names of accepted JWT issuers. This list must include a given IdP, or the cluster will reject JWTs issued by it.
+
+1. `server.jwt_authentication.audience`
+	
+	The name of your cluster as specified by the IdP, or a comma-separated list of such names. One of the audience names must match the `audience` field with which your IdP will generate JWT formatted auth tokens.
+
+## Authenticate to your cluster with your JWT token
+
+To provision SQL cluster access for service accounts, you must provision OIDC or SAML tokens. There are many ways to do this, which are beyond the scope of this tutorial.
+
+For example, your Google Cloud Platform organization can serve as IdP by issuing OIDC auth tokens, as described here in the [GCP docs on issuing tokens to service accounts](https://cloud.google.com/iam/docs/create-short-lived-credentials-direct#sa-credentials-oidc). This [blog post](https://morgans-blog.deno.dev/sso-crdb-gcp) discussing using GCP-issued OIDC tokens to authenticate to CockroachDB.
+
+Once you have a valid JWT auth token (with `issuer` and `audience` matching the values [configured in your cluster settings](#configure-your-cluster-settings)) from your IdP, you may use it to connect to your cluster's SQL interface.
+
+{{site.data.alerts.success}}
+This example uses [`cockroach sql`](cockroach-sql.html), but you can use any SQL client that supports sufficiently long passwords.
+{{site.data.alerts.end}}
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+cockroach sql --url "postgresql://{SQL_USERNAME}:{JWT_TOKEN}@{CLUSTER_HOST}:26257?options=--crdb:jwt_auth_enabled=true" --certs-dir={CLUSTER_CERT_DIR}
+~~~
+
+~~~txt
+Welcome to the cockroach SQL interface...
+~~~
+
+## Support for tokens with SQL-incompatible subject names
```

Comment:
The comments I left for this section on the nits and the heading for the cloud page, also apply to this page.

Reply:
thank you!